hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/FileIndex.java

```diff
@@ -69,7 +70,7 @@ public class FileIndex implements Serializable {
   private final org.apache.hadoop.conf.Configuration hadoopConf;
   private final PartitionPruners.PartitionPruner partitionPruner; // for partition pruning
   private final ColumnStatsProbe colStatsProbe;                   // for probing column stats
-  private final int dataBucketHashing;                                   // for bucket pruning
+  private final Option<Functions.Function1<Integer, Integer>> dataBucket;                                   // for bucket pruning
```

Comment:
Can we use Java function, can we align the indentation of the comments:

```java
  // for partition pruning
  // for probing column stats
  // for bucket pruning
```

Reply:
dataBucket -> dataBucketFunc

Reply:
chenged